### PR TITLE
develop Initial commit for the Algorythms  project.

### PR DESCRIPTION
**Pull request by alexandrulungoci**

### CODE REVIEW

File: src/com/sdacademy/algoritmi/tema1/AlDoileaCelMaiMic.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+public class AlDoileaCelMaiMic {
+
+    public static void main(String[] args) {
+
+        int[] input = {1, 3, 5, 2, 0};
+        bubbleSort(input);
+    }
+
+    static void bubbleSort(int[] input) {
+
```
Comment (gabrielionescu) [inline src/com/sdacademy/algoritmi/tema1/AlDoileaCelMaiMic.java]: You don't need to leave so many empty lines!

File: src/com/sdacademy/algoritmi/tema1/AlDoileaCelMaiMic.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        afisareArray(input);
+
+        int length = input.length;
+
+        int temp = 0;
```
Comment (gabrielionescu) [inline src/com/sdacademy/algoritmi/tema1/AlDoileaCelMaiMic.java]: You should extract the finding of the second smallest number into a method .

**Comment by gabrielionescu** [comment]
You can merge this in master, but **please take a  look at the code review improvements.**

**Comment by alexandrulungoci** [comment]
1. Daca facem modificari dupa ce am dat pull request reluam procedura
(commit, push, pull)?
2. Daca cream un nou package  intr-un proiect existent, facem iar pasii
(commit, push, pull) pentru noua clasa?

**Pull request merged**